data/notices.json

```diff
@@ -295,6 +295,36 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "Python users using Schedule class get synth-time TypeError for mismatched types",
```

Comment:
perhaps we should highlight python users only more given that this is going to blast non python users